stint/literals_stint.nim

```diff
@@ -34,12 +40,12 @@ macro make_mixed_types_ops(op: untyped, ResultTy: untyped, sign: static[Signedne
                     )
 
     result.add quote do:
-      proc `op`*[bits: static[int]](a: StUint[bits], b: `intLit`): `ResultTy` {.inline.}=
+      proc `op`*[bits: static[int]](a: StUint[bits], b: `uintLit`): `ResultTy` {.inline.}=
```

Comment:
what happens if it's out of range for the type? it should Defect (runtime) or error (compile-time) just like nim integers

Reply:
right, will add more tests to this.